sdk/ml/azure-ai-ml/azure/ai/ml/entities/_datastore/one_lake.py

```diff
@@ -0,0 +1,107 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=protected-access,no-member
+
+from pathlib import Path
+from typing import Any, Dict, Optional, Union
+
+from azure.ai.ml._restclient.v2023_04_01_preview.models import OneLakeDatastore as RestOneLakeDatastore, Datastore as DatastoreData, DatastoreType, OneLakeArtifact
+from azure.ai.ml._schema._datastore.one_lake import OneLakeSchema
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY, TYPE
+from azure.ai.ml.entities._credentials import NoneCredentialConfiguration, ServicePrincipalConfiguration
+from azure.ai.ml.entities._datastore.datastore import Datastore
+from azure.ai.ml.entities._datastore.utils import from_rest_datastore_credentials
+from azure.ai.ml.entities._util import load_from_dict
+
+
+class OneLakeDatastore(Datastore):
+    """OneLake datastore that is linked to an Azure ML workspace.
+
+    :param name: Name of the datastore.
+    :type name: str
+    :param artifact: OneLake Artifact. Only LakeHouse artifacts are currently supported.
+    :type artifact: OneLakeArtifact
```

Comment:
We're working to improve documentation standards across the SDK. One way we're doing that is by making sure documentation is linked together. Can you replace the types here with their full paths in this format: 

```
OneLakeArtifact -> ~azure.ai.ml.entities.OneLakeArtifact
ServicePrincipalConfiguration -> ~azure.ai.ml.entities.ServicePrincipalConfiguration
NoneCredentialConfiguration -> ~azure.ai.ml.entities.NoneCredentialConfiguration
```

Reply:
Thanks - updated!